.circleci/config.yml

```diff
@@ -581,19 +591,23 @@ jobs:
           key: v2-mix-specs-cache-{{ .Branch }}-{{ checksum "mix.lock" }}
       - run:
           name: Print watcher logs
-          command: docker-compose -f docker-compose.yml -f ./priv/cabbage/docker-compose-reorg.yml -f ./priv/cabbage/docker-compose-cabbage.yml logs --follow watcher
+          command: docker-compose -f docker-compose.yml -f ./priv/cabbage/docker-compose-reorg.yml -f ./priv/cabbage/docker-compose-2-specs.yml logs --follow watcher
```

Comment:
how do we name this better, `docker-compose-2-specs` is confusing I think...

Reply:
@InoMurko it means the docker-compose version 2. the childchain users version 3. what do you think about `docker-compose-version-2-specs`?

Reply:
oh, okay... that's the reason then. perhaps there's two things:
- should we name them by functionality?
- should we wrap these into makefile recipes?

Reply:
@InoMurko I moved to makefile. It doesn't have any functionality, it only overrides 2 env variables